packages/babel-cli/src/babel/index.js

```diff
@@ -163,7 +163,7 @@ commander.option(
   "--include-dotfiles",
   "Include dotfiles when compiling and copying non-compilable files",
 );
-commander.option("-q, --quiet", "Don't log anything");
+commander.option("-v, --verbose", "Log everything");
```

Comment:
Also, what do you think about the alias – should it be `-v` or maybe no alias at all? At least it doesn't collide with anything right now.

Reply:
Oh, havent spotted that yesterday - I think `verbose` is not like the most wanted/common thing and it might not deserve alias just yet, especially that `-v` is most commonly used for `version` in other tools.

Reply:
removed it